### ISSUE TITLE
IE9 creates mouse events with pageX and pageY set to 0.

### ISSUE THREAD

**Issue by Quentie** [joscha/eventr#1]
Thanks for your nice lib. Just noticed that there is an issue creating mouseEvents in IE9. I patched my eventr.js file but maybe you could add it directly to Eventr?

Code is taken from jquery-ui: https://github.com/jquery/jquery-ui/blob/e2d9b02c56ee92fa92913b451598b59a385db609/tests/jquery.simulate.js 

```
if (navigator.appVersion.indexOf("MSIE 9") != -1) {
    // IE9 creates events with pageX and pageY set to 0.
    // Trying to modify the properties throws an error,
    // so we define getters to return the correct values.
    if ( evt.pageX === 0 && evt.pageY === 0 && Object.defineProperty ) {
        eventDoc = evt.relatedTarget.ownerDocument || document;
        doc = eventDoc.documentElement;
        body = eventDoc.body;

        Object.defineProperty( evt, "pageX", {
            get: function() {
                return evt.clientX +
                    ( doc && doc.scrollLeft || body && body.scrollLeft || 0 ) -
                    ( doc && doc.clientLeft || body && body.clientLeft || 0 );
            }
        });
        Object.defineProperty( evt, "pageY", {
            get: function() {
                return evt.clientY +
                    ( doc && doc.scrollTop || body && body.scrollTop || 0 ) -
                    ( doc && doc.clientTop || body && body.clientTop || 0 );
            }
        });
    }
}
```

**Comment by joscha** [comment]
@Quentie cool! do you mind opening a pull request?

**Comment by Quentie** [comment]
Sure here you go: https://github.com/joscha/eventr/pull/2 
Actually it has been a while since I saw coffee code, but the resulting javascript looks fine to me ;)

**Comment by joscha** [comment]
closed via #2
